tests/integration/test_download.py

```diff
@@ -165,3 +166,18 @@ def test_above_threshold_for_nonseekable_fileobj(self):
                 self.bucket_name, '20mb.txt', NonSeekableWriter(f))
             future.result()
         assert_files_equal(filename, download_path)
+
+    @skip_if_windows('Windows does not support UNIX special files')
+    def test_download_to_special_file(self):
+        transfer_manager = self.create_transfer_manager(self.config)
+        filename = self.files.create_file_with_size(
+            'foo.txt', filesize=1024 * 1024)
+        self.upload_file(filename, '1mb.txt')
+        future = transfer_manager.download(
+            self.bucket_name, '1mb.txt', '/dev/null')
+        try:
+            future.result()
+        except Exception as e:
```

Comment:
I wonder if it is an appropriate assumption that we have permissions to write to /dev/null or if I should be creating my own special file like a fifo or socket?